packages/daemon/src/app/controller/job.ts

```diff
@@ -60,15 +61,18 @@ export class JobController {
         sse.finish();
       }
     } else {
-      try {
-        const plugins = await this.pipelineService.installPlugins(job, cwd, pyIndex);
+      const run = async () => {
```

Comment:
Why do you create this function and it gets called once.

Reply:
Moved into `nextTick`